src/siotls/examples/simple_client.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from socket import socket
+from siotls import TLSConfiguration, TLSConnection
+from siotls.utils import hexdump
+
+
+logger = logging.getLogger(__name__)
+
+def connect(host, port):
+    server = socket()
+    server.connect((host, port))
+    logger.info("connected to %s port %s", host, port)
+
+    config = TLSConfiguration('client')
+    conn = TLSConnection(config)
+    conn.initiate_connection()
```

Comment:
would config before opening the socket
so that it is easier to try/finally client.close